easy_my_coop/views/subscription_request_view.xml

```diff
@@ -106,12 +106,13 @@
                <field name="name"/>
                <separator/>

```

Comment:
@remytms could you also add a filter on the cooperator ?

Reply:
I will do it. :)